### DISCUSSION TITLE
simulate a radiator in FDS

### DISCUSSION THREAD

**Discussion by gesamarie** [firemodels/fds#12640] [Q&A]
Hello everyone,
I am trying to do a "Heating Load" Simulation.
I calculated the required **heating load** for a room I modeled in PyroSim. In my case, the heating load is 983 W.
How exactly do implement a radiator in FDS which has 983W?
Through some research I estimated that the surface temperature has to be 60°C. With the formula of the Heat-Transfer-Rate ([https://en.wikipedia.org/wiki/Heat_transfer_coefficient]) I figured out, that the Heat-Transfer-Coefficient has to be 17,31 W/(m^2K).

My question is, what is the correct way to implement the radiator into my model? I have implemented 3 different approaches and I am not sure, which is the correct way to implement it.

Version 1: Set surface temperature of the radiator to 60 °C 
`&SURF ID='Radiator_temp', TMP_FRONT=60.0/`

Version 2: Set surface temperature of the radiator to 60°C and also define the specific heat transfer coefficient as 17,31 W/(m^2*K)
`&SURF ID='Radiator_HTC_temp', HEAT_TRANSFER_COEFFICIENT=17.31, TMP_FRONT=60.0/`

Version 3: Set the specific heat transfer coefficient as 17,31 W/(m^2*K)
`&SURF ID='Radiator_HTC', HEAT_TRANSFER_COEFFICIENT=17.31/`

Thank you in advance

Best Regards
Gesa

**Comment by mcgratta** [comment]
Version 2. Note that there will also be some thermal radiation emitted by the object, but a very small amount.

**Reply by gesamarie** [reply]
Thank you for the answer.
That would have been the version where I would have guessed, that it is not correct, because the temperature for version 2 is alot higher than in the other versions. 
It definetly makes sense, that the temperature for version 2 is higher than the temperature for versions 1 and 3. But it also differs the most from the expected ambient temperatur.

**Comment by mcgratta** [comment]
How do you know that the temperatures are too high? How are you verifying your approach?

**Reply by gesamarie** [reply]
I have looked over the results again and the temperatures are not " too high". We have a norm, with which one can calculate how high the needed power is to maintain a room temperature with given U-values for the walls and an outside temperature. We suspected, that the estimated calculated power is too high. And our assumption seems to be true, as the average temperature in the room is approx. 23 °C and not 20°C.

**Comment by drjfloyd** [comment]
You have convective and radiative transfer to the interior surface of the room from the gas in the room and then convective and radiative transfer from the exterior surface to the ambient. To know that the temperatures are too high you will need to know the wall front and back side temperature, front and bcak side heat transfer coefficient, and wall adjacent gas temperature for every wall cell in the domain. If you have not output these but are only assuming values, this could easily lead to errors in what you think the room temperature should be.

**Reply by gesamarie** [reply]
Thank you for the reply. I have extracted some of these values ( I was about to look at them) but I will extract all of these values. That is a good idea because then I have values I could compare to my calculations.

**Comment by johodges** [comment]
I would check the math on your calculations. A heat transfer coefficient of 17 W/m2-K is pretty high for a delta T of ~40 C (1.31 [T-Tinf]^(1/3) = 5 W/m2-K] - Rough approximation of a vertical plate h).

**Reply by gesamarie** [reply]
I used the formula $\dot {Q}=hA(T_2-T_1)$ to calculate the heat-transfer-coefficient **h [W/m^2K]**.
    $\dot {Q}$: Heat transfer rate (W),  A: surface area where the heat transfer takes place (m²), $T_{2}$: temperature of the surrounding fluid (K), $T_{1}$: temperature of the solid surface (K)

I have used $T_1$= 333,15K =60°C, $T_2$= 293,15K=20°C, A=1,42m² (The Radiator has the dimensions 0,5m*1,1m*0,1m),  $\dot {Q}$=983W

By using the formula above I get the result $h=\frac{\dot {Q}}{A \cdot \Delta T}= 17,31 W/m^2K$.
Maybe the value seems too high, because the radiator has a small size and the needed Power is fairly high.

**Comment by drjfloyd** [comment]
If you are looking to ensure you always have 983 W, you could specifcy a fixed net heat flux at the surface. See the User's Guide for details.

**Reply by gesamarie** [reply]
That is probably the smarter option to choose because it makes it possible to ensure, that 983W are released. I overlooked the option for surfaces. Thank you, I will look it up in the UserGuide
